truffle.js

```diff
@@ -18,6 +23,7 @@ module.exports = {
       },
     },
   },
+  contracts_directory: process.env.COVERAGE ? covContractsDir : regContractsDir,
```

Comment:
Makes sure `truffle migrate` doesn't recompile all the contracts.

(tbh I'm not completely sure this is necessary either - might be relic of initial confusion)